torch/nn/_functions/rnn.py

```diff
@@ -210,14 +214,17 @@ def backward_extended(self, grad_output, grad_hy):
             grad_hx)
 
         if any(self.needs_input_grad[1:]):
-            grad_weight = [tuple(w.new().resize_as_(w) for w in layer_weight) for layer_weight in weight]
```

Comment:
This `zero_` is unnecessary (it's been removed, but you probably didn't notice that when rebasing).